blocks/columns/columns.js

```diff
@@ -143,9 +143,7 @@ export default function decorate(block) {
 
           // remove empty paragraphs
           col.querySelectorAll('p').forEach((p) => {
-            if (!p.querySelector('a')) {
-              p.remove();
-            }
+            if (!p.querySelector('a') && p.innerHTML === '') p.remove();
```

Comment:
The first condition looks like a subset of the second one. Do we need both of them?

Reply:
Yes, Nice Catch 😄